lib/rdoc/generator/template/darkfish/css/rdoc.css

```
@@ -579,8 +579,8 @@ main .method-detail {
   margin-bottom: 2.5em;
 }
 
-main .method-detail:target {
-  margin-left: -10px;
+main .method-detail:target .method-heading {
+  margin-left: 10px;
```

```suggestion
  margin-left: -10px;
```
(to revert the change)